src/requestforattestation/RequestForAttestation.ts

```diff
@@ -52,15 +77,167 @@ function generateHashTree(contents: object): object {
   return result
 }
 
-function verifyClaimerSignature(rfa: RequestForAttestation): boolean {
-  return verify(rfa.rootHash, rfa.claimerSignature, rfa.claim.owner)
+function verifyClaimerSignature(reqForAtt: RequestForAttestation): boolean {
+  return verify(
+    reqForAtt.rootHash,
+    reqForAtt.claimerSignature,
+    reqForAtt.claim.owner
+  )
 }
 
 function getHashRoot(leaves: Uint8Array[]): Uint8Array {
   const result = u8aConcat(...leaves)
   return hash(result)
 }
 
+/**
+ *  Compresses an nonce and hash from a [[ClaimHashTree]] or [[RequestForAttestation]] properties.
+ *
+ * @param nonceHash A hash or a hash and nonce object that will be sorted and stripped for messaging or storage.
+ *
+ * @returns An object compressing of a hash or a hash and nonce.
+ */
+
+export function compressNonceAndHash(
+  nonceHash: NonceHash
+): CompressedNonceHash {
+  return [nonceHash.hash, nonceHash.nonce]
+}
+
+/**
+ *  Decompresses an nonce and hash from a [[ClaimHashTree]] or [[RequestForAttestation]] properties.
+ *
+ * @param nonceHash A compressesd a hash or a hash and nonce array that is reverted back into an object.
+ *
+ * @returns An object compressing of a hash or a hash and nonce.
+ */
+
+function decompressNonceAndHash(nonceHash: CompressedNonceHash): NonceHash {
+  // Need to fix
```

Comment:
What needs to be fixed?

Reply:
I left an `any` and we have cleared it already.